test_programs/compile_success_empty/comptime_module/src/main.nr

```diff
@@ -1,10 +1,45 @@
+#[outer_attribute]
 mod foo {
+    #![some_attribute]
```

Comment:
I wish we had a way of differentiating attributes which are just annotations from ones that we should run as functions. I spent a bit too long searching for `out_attribute` and `some_attribute` functions.